XenAdmin/Wizards/PatchingWizard/PatchingWizard_SelectPatchPage.cs

```diff
@@ -126,18 +138,27 @@ public override string HelpID
         public override void PageLoaded(PageLoadedDirection direction)
         {           
             base.PageLoaded(direction);
-            RefreshListButton.Enabled = true;
             Updates.CheckForUpdatesStarted += CheckForUpdates_CheckForUpdatesStarted;
             Updates.CheckForUpdatesCompleted += CheckForUpdates_CheckForUpdatesCompleted;
+            Updates.RestoreDismissedUpdatesStarted += Updates_RestoreDismissedUpdatesStarted;
          
             if (direction == PageLoadedDirection.Forward)
             {
+                //if any connected host is licensed for automatic updating
+                bool autoUpdatePossible = ConnectionsManager.XenConnectionsCopy.Any(c => c != null && c.Cache.Hosts.Any(h => !Host.RestrictBatchHotfixApply(h)));
+               
+                labelWithAutomatic.Visible = automaticOptionLabel.Visible = AutomaticRadioButton.Visible = autoUpdatePossible;

```

Comment:
Remove this from here because of the conneciton is slow the automatic button may be visible  to start with and then disappear.